.github/workflows/test.yml

```diff
@@ -17,6 +17,7 @@ jobs:
 
     steps:
       - uses: actions/checkout@v3
+      - uses: actions-rust-lang/setup-rust-toolchain@v1
```

Comment:
As far as I can tell, `actions-rust-lang` is owned by neither GitHub nor the Rust project. Do you know if either of these parties is working on an official action? If not, either
* is `v1` a tag we can audit or something, which will not change until we change this file?
* how annoying is it to use rustup instead?

Reply:
We can probably(?) do rustup, I copied this from #58